src/SOS/Strike/sigparser.cpp

```diff
@@ -122,8 +121,18 @@ HRESULT SigParser::SkipExactlyOne()
 //
 HRESULT
 SigParser::SkipMethodHeaderSignature(
-    uint32_t * pcArgs)
+    uint32_t * pcArgs, bool skipReturnType /*= true*/)
 {
+    CONTRACTL
```

Comment:
Why is this not in the shared component? Can you please not this in the shared readme (and any other files that might be needed.)

Reply:
Added note in shared/README.txt.  In the runtime it is part of utilcode which SOS can't include.